src/table.js

```diff
@@ -699,18 +699,18 @@ class Table {
    *
    * @throws {error} If a name is not provided.
    *
-   * @param {string} name The family name.
+   * @param {string} id The family unique identifier.
    * @returns {Family}
    *
    * @example
    * const family = table.family('my-family');
    */
-  family(name) {
-    if (!name) {
+  family(id) {
```

Comment:
Please also change any uses of `family()` to only get the id instead of the full name.